packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -29,18 +29,15 @@ type FileDep = {
 
 type VitePreloadErrorEvent = Event & { payload: Error }
 
-/**
- * A flag for injected helpers. This flag will be set to `false` if the output
- * target is not native es - so that injected helper logic can be conditionally
- * dropped.
- */
-export const isModernFlag = `__VITE_IS_MODERN__`
+// Placeholder symbols for injecting helpers
+// TODO: obfuscate these in next major version
```

Comment:
```suggestion
```
I prefer not to add this TODO comment for now.

Reply:
Removed